submissions/Semka094/hooli-style-popup/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Jost:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap"
+          rel="stylesheet">
+    <title>Pop-up</title>
+</head>
+
+<body>
+
+<div class="menuContainer">
+    <nav>
+        <ul class="navWrapper">
+            <li class="navItem">
+                <a class="navLink" href="#">Mail</a>
+            </li>
+            <li class="navItem">
+                <a class="navLink" href="#">Images</a>
+            </li>
+            <li class="navItem icon">
+                <input class="popupBtnCheckbox" type="checkbox" id="popupBtn">
+                <label class="popupBtnLabel" for="popupBtn">
+                    <img class="navImg" src="./images/popup-button.png" alt="">
+                </label>
+
+                <div class="appsPopup">
+                    <div class="popupContainer">
+                        <ul class="itemList">
+                            <li>
+                                <a href="#" class="columnItem">
+                                    <img class="popupItemImg" src="./images/calendar.png" alt="Icon of calendar">
```

Comment:
What about the rest of the icons?

Reply:
Since you did not provide any specificity from the beginning (just a link to an article) on what is wrong with my alt, I assumed that the alt text itself should be more self-explicit. 

Now, I understand you want me to remove alts from ALL images as they are decorative.

Fixed.

Reply:
Look, you have a lot of icons implemented with `img` tag inside the links, I mention the features of `alt` attribute for icons and you fix it only for line with comment. Why you don't change `alt` for other similar elements?)

Reply:
I thought your comment is related only to menu images))
Thanks for the code review and patience.